lib/releasy.js

```diff
@@ -44,10 +43,10 @@ module.exports = function (opts) {
   config.npmFolder = opts.npmFolder
   config.dryRun = opts.dryRun
   config.quiet = quiet
-  config.unreleased = '## [Unreleased]'
+  config.unreleasedRaw = 'Unreleased'
+  config.unreleased = `## [${config.unreleasedRaw}]`
```

Comment:
Is this really necessary?

Reply:
not really, I did this because I expected to use the "Unreleased" string in the `addChangelogLinks` function but I see now that I didn't 😅